website/docs/r/cms_sitemonitor.markdown 

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Cloud Monitor"
+layout: "alicloud"
+page_title: "Alicloud: alicloud_cms_sitemonitor"
+sidebar_current: "docs-alicloud-resource-cms-sitemonitor"
+description: |-
+  Provides a resource to build a sitemonitor rule for cloud monitor.
+---
+
+# alicloud\_cms\_sitemonitor
+
+This resource provides a sitemonitor resource and it can be used to monitor public endpoints and websites.
+Details at https://www.alibabacloud.com/help/doc-detail/67907.htm
+
```

Comment:
Please add a note to point the available version : `Available in 1.72.0+`

Reply:
done